cfgov/v1/blocks.py

```diff
@@ -1,5 +1,8 @@
+from django.core.exceptions import ValidationError
 from wagtail.wagtailcore import blocks
 from django.utils.module_loading import import_string
+from django.utils.text import slugify
+from .util.util import get_unique_id
```

Comment:
Nitpicky style thing: [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) recommends grouping third-party imports together (Django & Wagtail) separated by a space by local imports (the util ones). Within each group we also tend to try to alphabetize the imports (e.g. `from django` before `from wagtail`).